lab_4_retrieval_w_clustering/main.py

```diff
@@ -81,6 +113,16 @@ def _calculate_bm25(self, tokenized_document: list[str]) -> Vector:
         Returns:
             Vector: BM25 vector for document.
         """
+        if not isinstance(tokenized_document, list) or not tokenized_document\
+                or not all(isinstance(token, str) for token in tokenized_document):
+            raise ValueError('Inappropriate type input argument or input argument is empty.')
+        vector = [0.0] * len(self._vocabulary)
+        bm_25 = calculate_bm25(self._vocabulary, tokenized_document, self._idf_values, 1.5, 0.75,
```

Comment:
rm default arguments

Reply:
without them test_calculate_bm25_ideal and VectorDBSearchEngineTest::test_retrieve_relevant_documents_ideal fail